kore/src/Kore/Strategies/Goal.hs

```diff
@@ -1064,27 +1045,23 @@ getDestination
     :: forall goal
     .  ToRulePattern goal
     => goal
-    -> Pattern Variable
-getDestination (toRulePattern -> RulePattern { right, ensures }) =
-    Pattern.withCondition right (Conditional.fromPredicate ensures)
+    -> RHS Variable
+getDestination (toRulePattern -> RulePattern { rhs }) = rhs
 
 makeRuleFromPatterns
     :: forall rule
     .  FromRulePattern rule
     => rule
     -> Pattern Variable
-    -> Pattern Variable
+    -> RHS Variable
     -> rule
-makeRuleFromPatterns ruleType configuration destination =
+makeRuleFromPatterns ruleType configuration rhs =
```

Comment:
Maybe this should get renamed to something like `makeRuleFromComponents`, although I'm not sure that is the best name. Do you or @ttuegel have any other ideas?

Reply:
I've renamed it as `configurationDestinationToRule` to match the naming of the "destructors" above